testenv-compose.yml

```diff
@@ -113,3 +113,6 @@ services:
       container: docker
       ION_NODE_NUM: 3
       MGR_NODE_NUM: 1
+      DTNMA_LOGLEVEL: debug
+      DUMPCAP_OUTFILE: /var/log/pcap/agent1.pcap
```

Comment:
Just realized that these agent log files should be 3 here and 2 above to deconflict. They use the same mount point.